components/style/values/computed/length.rs

```diff
@@ -473,4 +474,69 @@ pub type LengthOrNone = Either<Length, None_>;
 
 pub type LengthOrNumber = Either<Length, Number>;
 
+#[derive(Copy, Clone, PartialEq)]
```

Comment:
In order to define how to animate `MinLength`, I tried to copy the style used in `VerticalAlign`, which starts with `match (*self, *other)`. I got compiler errors because this meant moving the `MinLength`. I made the compiler error go away by defining `MinLength` to be `Copy`. Is this right? If not, how do I solve the compiler error for `Interpolate`?